designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -493,8 +493,11 @@ class DeploymentServiceImpl(
           case CompletedNormally(value) =>
             value
           case CompletedByTimeout(value) =>
+            logger.trace(s"Checking fetching scenario state ${processIdWithName.name} with policy $freshnessPolicy")
```

Comment:
Do we need the trace log if we already have logger.warn with same data in the next line?

Reply:
In fact tracing in HttpFlinkClient is enough.